### PR TITLE
start using local css file

### PR DESCRIPTION
**Pull request by dstoianov**
problems:
 -    `spring.jackson.property-naming-strategy=SNAKE_CASE` is not works any more, since we have `@EnableWebMvc` enabled

**Remark:** you can reach the API by this URL `http://localhost:8080/doc`

### CODE REVIEW

File: .github/workflows/maven-feature.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
+      - name: Maven Test
+        run: mvn -B clean test
```
Comment (VladimirYushkevich) [inline .github/workflows/maven-feature.yml]: you can remove "clean"....noting to clean in container first time

File: src/main/java/se/techinsight/urlshortener/UrlController.java [hunk 2]
```diff
@@ -21,47 +20,47 @@
 @Tag(name = "Url Controller", description = "API for working with URL Controller")
 @RestController
 @Validated
-@RequestMapping({"/api/url"})
+@RequestMapping(value = {"/api/url"})
 public class UrlController {
 
     @Autowired
     private UrlShortenerService service;
```
Comment (VladimirYushkevich) [inline src/main/java/se/techinsight/urlshortener/UrlController.java]: I would prefer constructor injection. If you have a lombok: @RequiredArgsConstructor + private final UrlShortenerService service;

File: src/main/java/se/techinsight/urlshortener/UrlController.java [hunk 3]
```diff
@@ -21,47 +20,47 @@
...
     @PostMapping(value = "/", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
     public UrlShortener create(@Valid @RequestBody OriginalUrlDto dto) {
         return service.save(dto);
     }
 
+    @Operation(summary = "Delete URL object by {shorten_key}")
+    @ResponseStatus(HttpStatus.ACCEPTED)
     @DeleteMapping(value = "/{shorten_key}", produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity delete(@PathVariable("shorten_key") @Size(min = 1, max = 6) String shortenKey) {
+    public Map<String, Object> delete(@PathVariable("shorten_key") @Size(min = 1, max = 6) String shortenKey) {
```
Comment (karmanov) [inline src/main/java/se/techinsight/urlshortener/UrlController.java]: Why do you return the map here? Is ACCEPTED status code will be not enough for the front-end to understand that operation finished successfully?

**Comment by karmanov** [comment]
LGTM, just 1 minor comment

File: src/main/java/se/techinsight/urlshortener/UrlController.java (hunk 3 above)
Comment (dstoianov) [reply src/main/java/se/techinsight/urlshortener/UrlController.java]: probably yes, but i wanted to show FE dev or consumer of this API that operation finished successfully, or it is bad practice, the response status 202 will be enough?

File: src/main/java/se/techinsight/urlshortener/UrlController.java (hunk 2 above)
Comment (dstoianov) [reply src/main/java/se/techinsight/urlshortener/UrlController.java]: ok, will do it

**Pull request merged**